pyproject.toml

```diff
@@ -69,7 +69,7 @@ sphinxext-opengraph = "^0.6.3"
 sphinx-click = "^4.4.0"
 sphinxcontrib-mermaid="^0.8.1"
 sphinx-togglebutton="^0.3.2"
-reuse = "^1.0.0"
+reuse = "^1.1.2"
```

Comment:
Is this update required to make this patch work?

I think we decided to keep changes in `pyproject.toml` and  `poetry.lock` to a minimum. Do we allow package updates by PR side-effect?

Reply:
**opinion:** Well, I think as there is a *proper* change here, it's okay to also update the dependencies...